docs/examples/frameworks/jax/jax-basic_example.ipynb

```diff
@@ -5,7 +5,7 @@
    "cell_type": "markdown",
```

Comment:
<blockquote>apply one hot encoding to them for training with <code>fn.one_hot</code></blockquote><p>Nitpick: It sounds a bit as if fn.one_hot was a way of training a network. How about something like "use fn.one_hot to apply one hot encoding to them for training"?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA/DALI/pull/5176/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Done